tests/pkcli/build_data/1.out/srv/host/v9.radia.run/srv/rsiviz/cmd

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I don't find this file to be useful to include in test data.

Reply:
1b36f4b